contracts/CompoundGovernor.sol

```diff
@@ -1,5 +1,186 @@
 // SPDX-License-Identifier: BSD-3-Clause
-pragma solidity 0.8.18;
+pragma solidity 0.8.26;
 
-// TODO: Implement new governance contract for Compound. This is just a placeholder to satisfy "forge fmt".
-contract CompoundGovernor {}
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {GovernorUpgradeable} from "@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol";
+import {GovernorVotesUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesUpgradeable.sol";
+import {GovernorCountingSimpleUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorCountingSimpleUpgradeable.sol";
+import {GovernorTimelockControlUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorTimelockControlUpgradeable.sol";
+import {TimelockControllerUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/TimelockControllerUpgradeable.sol";
+import {GovernorSettingsUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorSettingsUpgradeable.sol";
+import {GovernorPreventLateQuorumUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorPreventLateQuorumUpgradeable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {IVotes} from "@openzeppelin/contracts/governance/utils/IVotes.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+/// @title CompoundGovernor
+/// @author [ScopeLift](https://scopelift.co)
+/// @notice A governance contract for the Compound DAO.
+/// @custom:security-contact TODO: Add security contact
+contract CompoundGovernor is
+    Initializable,
+    GovernorVotesUpgradeable,
+    GovernorTimelockControlUpgradeable,
```

Comment:
We need to use `GovernorTimelockCompoundUpgradeable.sol` instead

Reply:
okay with doing it in a seperate PR as well

Reply:
^ defer to you @jferas

Reply:
I don't see a separate issue for it, so I'll add it in here

Reply:
Done: 08411a0149ba3f6e7165082e32462ed9ff778835